public/javascripts/modules/calculator.mjs

```diff
@@ -28,24 +28,97 @@ const addListeners = () => {
       Array.from(key.parentNode.children)
           .forEach(k => k.classList.remove('is-depressed'))
 
+      const calculate = (n1, operator, n2) => {
+        let result = '';
+
+        if (operator === 'add') {
+          result = parseInt(n1) + parseFloat(n2);
+        } else if (operator === 'subtract') {
+          result = parseInt(n1) + parseFloat(n2);
```

Comment:
In each `else if` block you are adding the two numbers. Just switch the symbols to `-`, `*`, and `/` when checking each operator.

Reply:
Good catch. serves me right from not proof reading after copying and pasting lol